src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/DateFormatIT.java

```diff
@@ -118,6 +119,22 @@ public void or() throws SqlParseException {
         );
     }
 
+
+    @Test
+    public void sortByDateFormat() throws IOException {
+        // Sort by expression in descending order, but sort inside in ascending order, so we increase our confidence
+        // that successful test isn't just random chance.
+
+        JSONArray hits =
+                getHits(executeQuery("SELECT all_client, insert_time " +
+                        "FROM " + TestsConstants.TEST_INDEX_ONLINE + "/online " +
```

Comment:
I suggest we do not specify the type of the documents, since it's already deprecated, and we'll have to remove that usage soon in the future anyway. So do not add the `"/online"` part.

Reply:
Followed existing approach, didn't know about deprecated /online, thanks.
Removed